app/gui/src/dashboard/components/AriaComponents/Inputs/Selector/Selector.stories.tsx

```diff
@@ -0,0 +1,84 @@
+import type { Meta, StoryObj } from '@storybook/react'
+import { z } from 'zod'
+
+import { Form } from '#/components/AriaComponents'
+import { userEvent, within } from '@storybook/test'
+import type { SelectorProps } from './Selector'
+import { Selector } from './Selector'
+
+// Schema for our form
+const schema = z.object({
+  plan: z.enum(['basic', 'pro', 'enterprise']),
+})
+
+type Props = SelectorProps<typeof schema, 'plan'>
+
+export default {
+  title: 'Components/AriaComponents/Inputs/Selector',
+  component: Selector,
+  parameters: {
+    layout: 'centered',
+  },
+  render: (args) => <Selector {...args} />,
+  args: {
+    name: 'plan',
+    items: ['basic', 'pro', 'enterprise'],
+  },
+  decorators: [
+    (Story, context) => (
+      <Form schema={schema} className="w-96" defaultValues={{ plan: 'basic' }}>
```

Comment:
@somebody1234 I think this could be a reusable decorator, wdyt? Where should we put it?

Reply:
"this" as in...?

Reply:
oh i mean which part is reusable?

Reply:
``` typescript
(Story, context) => (
  <Form schema={schema} className="w-96" defaultValues={{ plan: 'basic' }}>
    {Story(context)}
  </Form>
),
```

This one. Can be reused across all form components

Reply:
line `formDecorator(formProps)`